attractions.html

```diff
@@ -0,0 +1,82 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Filmopolis Attractions</title>
+<!-- link css -->
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    
+</head>
+
+
+<body>
+<div id="site_wrap">
```

Comment:
We should use the same structure as the [index.html page](https://github.com/zhorzhz/park/blob/main/index.html), to be consistent.

```html
<body>
    <header>
    </header>
    <main>
    </main>
    <footer>
    </footer>
</body>
```

Reply:
Makes sense, I fixed the tags :)